src/solvers/dgsem_p4est/containers.jl

```diff
@@ -435,8 +435,9 @@ cfunction(::typeof(init_surfaces_iter_face), ::Val{3}) = @cfunction(init_surface
 # Function barrier for type stability
 function init_surfaces_iter_face_inner(info, user_data)
   @unpack interfaces, mortars, boundaries = user_data
+  elem_count = unsafe_load(info).sides.elem_count
```

Comment:
Just a side note (feel free to close): This is one of the more annoying changes with using Clang.jl: Now you have to "know" (or look up) for each nested field whether it is stored as a pointer or as a value :-/

Reply:
Yes, definitely.

Reply:
Maybe we can (have someone) come up with a method such as `pointer_load(info, sides, elem_count)` that will just do the Right Thing ™️ and dereference whatever is necessary to get to the final item...

Reply:
I added it to the list in https://github.com/trixi-framework/P4est.jl/issues/54, too :+1:

Reply:
🙏